src/mpyl/reporting/targets/jira.py

```diff
@@ -89,7 +89,7 @@ def from_config(config: dict):
         if not jira_config:
             raise KeyError('jira section needs to be defined in mpyl_config.yml')
         return JiraConfig(site=jira_config['site'], user_name=jira_config['userName'], password=jira_config['password'],
-                          ticket_pattern=re.compile(jira_config.get('ticketPattern', '[A-Za-z]{3,}-\\d+')),
+                          ticket_pattern=re.compile(jira_config.get('ticketPattern', '[A-Za-z]{2,}-\\d+')),
```

Comment:
Can we add a unit test for this?

Reply:
The tests are there, just need to be linked up with the default config:
https://github.com/Vandebron/mpyl/blob/97a5d872414aae2d59c51db7e6a609c9acd8c5b9/tests/reporting/targets/test_jira.py#L45-L54